modules/images-configuration-registry-mirror.adoc

```diff
@@ -77,17 +77,32 @@ spec:
   repositoryDigestMirrors:
   - mirrors:
     - example.io/example/ubi-minimal <1>
-    source: registry.access.redhat.com/ubi8/ubi-minimal <2>
-  - mirrors:
-    - example.com/example/ubi-minimal
-    source: registry.access.redhat.com/ubi8/ubi-minimal
+    - example.com/example/ubi-minimal <2>
+    source: registry.access.redhat.com/ubi8/ubi-minimal <3>
   - mirrors:
     - mirror.example.com/redhat
-    source: registry.redhat.io/openshift4 <3>
+    source: registry.redhat.io/openshift4 <4>
+  - mirrors:
+    - mirror.example.com
+    source: registry.redhat.io <5>
+  - mirrors:
+    - mirror.example.net/image
+    source: registry.example.com/example/myimage <6>
+  - mirrors:
+    - mirror.example.net
+    source: registry.example.com/example <7>
+  - mirrors:
+    - mirror.example.net/registry-example-com
+    source: registry.example.com <8>
 ----
 <1> Indicates the name of the image registry and repository.
-<2> Indicates the registry and repository containing the content that is mirrored.
+<2> Indicates multiple mirror repositories for the each target repository. If a one mirror is down, the target repository can use another mirror.
 <3> You can configure a namespace inside a registry to use any image in that namespace. If you use a registry domain as a source, the `ImageContentSourcePolicy` resource is applied to all repositories from the registry.
+<4> Indicates the name of the mirror.
+<5> If you configure the registry name in that namespace, the `ImageContentSourcePolicy` resource is applied to all repositories from a source registry to a mirror registry.
+<6> Pulls the image `mirror.example.net/image@sha256:...`.
+<7> Pulls the image `myimage` in the source registry namespace from the source mirror `mirror.example.net/myimage@sha256:...`.
+<8> Pulls the image `registry.example.com/example/myimage` from the source registry `mirror.example.net/registry-example-com/example/myimage@sha256:...`. The `ImageContentSourcePolicy` resource is applied to all repositories from a source registry to a mirror registry `mirror.example.net/registry-example-com`.
```

Comment:
s/the source registry/ the mirror registry

Reply:
Addressed